dev/src/steamworks.cpp

```diff
@@ -561,4 +770,250 @@ nfr("p2p_receive_messages", "", "", "S]S]", "receive messages from all"
         Push(sp, client_vec);
     });
 
+nfr("lobby_create", "max_members", "I", "B", "TODO", [](StackPtr &, VM &, Value &max_members) {
+    auto ok = false;
+    #ifdef PLATFORM_STEAMWORKS
+        if (steam) {
+            ok = steam->CreateLobby(max_members.intval());
+        }
+    #endif  // PLATFORM_STEAMWORKS
+    return Value(ok);
+});
+
+nfr("lobby_join", "steam_id", "I", "B", "TODO", [](StackPtr &, VM &, Value &steam_id) {
+    auto ok = false;
+    #ifdef PLATFORM_STEAMWORKS
+        if (steam) {
+            ok = steam->JoinLobby(SteamIDFromValue(steam_id));
+        }
+    #endif  // PLATFORM_STEAMWORKS
+    return Value(ok);
+});
+
+nfr("lobby_leave", "steam_id", "I", "B", "TODO", [](StackPtr &, VM &, Value &steam_id) {
+    auto ok = false;
+    #ifdef PLATFORM_STEAMWORKS
```

Comment:
This pattern is getting repetitive.. possibly a macro?

Reply:
Yeah, it's kind of a pain. I'll take a look